main/src/resources/benchmark/BenchmarkSet.flix

```diff
@@ -0,0 +1,126 @@
+namespace BenchmarkSet {
+
+    ///
+    /// create a set of integers and compute the size of the result.
+    ///
+    @benchmark
+    pub def benchmark01(): Bool = Set.size(List.toSet(List.range(0, 75))) == 75
```

Comment:
Obviously it takes time to construct a list and then convert it to a set. You may want to add a Set.range function to build the set directly.

Reply:
Sounds like a good idea.
> Otherwise I don't see how it can be slower for large enough n.
Increasing the number of elements helped, it is faster (as per benchmark) for n > ~125
My initial benchmark only inserted 75 elements to limit execution time.

Reply:
You might want to try to insert elements both in increasing, decreasing, and random order.

Reply:
Yep, i'll take another look tomorrow at caffeine peak time.